eclipse-collections/src/main/java/org/eclipse/collections/impl/bag/sorted/mutable/UnmodifiableSortedBag.java

```diff
@@ -560,4 +564,54 @@ protected Object writeReplace()
     {
         return this.getSortedBag().selectUnique();
     }
+
+    @Override
+    public <V> MutableObjectLongMap<V> sumByInt(Function<? super T, ? extends V> groupBy, IntFunction<? super T> function)
```

Comment:
Same question as UnmodifiableBag.

Reply:
Same reason as above. In order to optimize these methods, I had to add concrete implementations for these classes as they do not extend from `AbstractMutableBagIterable`.

Reply:
Same question regarding test coverage

Reply:
Same response as above.